modules/core/arrow-annotations-processor/models/src/main/java/arrow/ap/objects/iso/IsoCompanion.kt

```diff
@@ -0,0 +1,8 @@
+package arrow.ap.objects.iso
+
+import arrow.core.Option
+import arrow.optics.OpticsTarget
+import arrow.optics.optics
+
+@optics([OpticsTarget.ISO])
+data class IsoCompanion(val field: String, val nullable: String?, val option: Option<String>)
```

Comment:
Companion object?

Reply:
Test model to test for a compiler error. https://github.com/arrow-kt/arrow/pull/825/files#diff-8bcf2e9e0361c1d797accab0aede2539R32